### DISCUSSION TITLE
Workshop task repeat button

### DISCUSSION THREAD

**Discussion by willkydd** [DFHack/dfhack#5007] [Ideas]
It would be nice to have a way to build the same thing as before from workshops in the premium version of DF. For example let's say click add new task then rock then table. If I want another table I need to do all that again, with the mouse. It would be nice I could just hit r from the top level tasks context and re-add the last task.

This is similar and related to but different from this idea: https://github.com/DFHack/dfhack/discussions/4252

**Comment by unboundlopez** [comment]
I really like that idea. After I finish my current project I’ll give it a try.
